packages/@uppy/status-bar/src/style.scss

```diff
@@ -55,7 +55,7 @@
 .uppy-StatusBar:not([aria-hidden=true]).is-waiting {
   background-color: $white;
   height: 65px;
-  border-top: 1px solid $gray-200;
+  border-top: 1px solid #eaeaea;
 }
```

Comment:
Could we stick to the variables for colors, maybe we should be using a different one? Not sure about the reason for this change.

Reply:
@arturi, I agree. It was a change from the merge with master, I'm not sure why it conflicted. I'll change it to `$gray-200`, these are the same colors.